CHANGELOG.rst

```diff
@@ -18,6 +18,31 @@ Unreleased
 Added
 _____
 
+[0.2.6] - 2017-08-07
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Upgrade version of django-celery.
+
+[0.2.5] - 2017-08-03
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Django 1.11 compatibility
+
+[0.2.4] - 2017-06-20
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Add management command to fix djcelery tables.
+
+[0.2.1] - 2017-05-22
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Add ChordableDjangoBackend and testing.
```

Comment:
I can't believe that the dev that made these commits forgot to update the changelog. What a scrub!

(But really, thanks for updating this!)